sklearn/cross_validation.py

```diff
@@ -1213,9 +1213,9 @@ def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
           - An object to be used as a cross-validation generator.
           - An iterable yielding train/test splits.
 
-        For integer/None inputs, if ``y`` is binary or multiclass,
-        :class:`StratifiedKFold` used. If the estimator is a classifier
-        or if ``y`` is neither binary nor multiclass, :class:`KFold` is used.
+        For integer/None inputs, if ``y`` is binary or multiclass or if the
```

Comment:
I feel neither the old nor the new is good. If the classifier is a classifier the data is multi-label, it uses standard KFold, right?